sql-cli/conftest.py

```diff
@@ -2,7 +2,6 @@
 from pathlib import Path
 
 import pytest
-
```

Comment:
Wondering if this blank line separator for local package should get deleted in this file and all files below for `sql-cli` directory. What do you think?

Reply:
not sure -- it is auto-generated and doesn't affect runtime so 🤷